deep_translator/google_trans.py

```diff
@@ -74,7 +98,8 @@ def is_language_supported(self, *languages):
         for lang in languages:
             if lang != 'auto' and lang not in self._languages.keys():
                 if lang != 'auto' and lang not in self._languages.values():
-                    raise LanguageNotSupportedException(lang)
+                    if (self.is_secondary(lang)==0):
```

Comment:
you are checking here if language is secondary.  So you don't have to check again in the map_to_languages function

Reply:
done